crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -303,55 +303,56 @@ pub enum MoveFunctionArgType {
 }
 
 #[derive(Serialize, Deserialize, Debug, JsonSchema)]
-pub struct TransactionEffectsResponse {
+pub struct SuiTransactionEffectsResponse {
     pub certificate: SuiCertifiedTransaction,
     pub effects: SuiTransactionEffects,
     pub timestamp_ms: Option<u64>,
+    pub parsed_data: Option<SuiParsedTransactionResponse>,
```

Comment:
If this is only useful to the Sui CLI, I think we can remove this in future PR after my client side object state PR is done. Converting effect to specialised response type on client side should be cheap after that.